modules/nf-core/seqkit/fq2fa/main.nf

```diff
@@ -0,0 +1,49 @@
+process SEQKIT_FQ2FA {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::seqkit=2.5.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/seqkit:2.5.0--h9ee0642_0':
+        'biocontainers/seqkit:2.5.0--h9ee0642_0' }"
+
+    input:
+    tuple val(meta), path(fastq)
+
+    output:
+    tuple val(meta), path("*.fa.gz"), emit: fasta
```

Comment:
What do you think about adding support for zstandard compression (`.fa.zst`) here?

Reply:
looks good